crates/rolldown/tests/rolldown/issues/4472/artifacts.snap

```diff
@@ -14,24 +14,14 @@ export * from "ext-folder"
 
 // HIDDEN [rolldown:runtime]
```

Comment:
Now we can generate almost identical code to Rollup, except for the runtime helper, since it has been referenced before `included_statements`. Luckily, the code can be removed after minification.